webapp/components/ContributionForm/ContributionForm.spec.js

```diff
@@ -201,7 +220,12 @@ describe('ContributionForm.vue', () => {
             content: 'auf Deutsch geschrieben',
             language: 'de',
             image,
-            categories: [{ id: 'cat12', name: 'Democracy & Politics' }],
+            categories: [
+              {
+                id: 'cat12',
+                name: 'Democracy & Politics',
+              },
+            ],
```

Comment:
@ogerly try not to touch unrelated files. First, it makes revieweing your PR difficult. Second and more important, you will "own" the code. This will make it difficult to understand the code, because `git blame` will tell me that you are the author, but you just changed indentation.

To fix this: Run`git checkout origin/master -- path/to/unrelated/files`.

Reply:
yes, i'm sorry. 
i have a lint problem in my editor. 
when i edit a page that i later restore to its original state. there is still a change in the git status due to the lint. 
should i just turn it off in my editor ? 
![Bildschirmfoto vom 2019-07-10 06-51-51](https://user-images.githubusercontent.com/1324583/60941430-6d7fa780-a2df-11e9-856d-492757060dc5.png)